Azure/sonic-buildimage
# [rules/sonic-utilities.mk]: Add sonic_yang_models as dep to sonic utils.

Since we can not refer a directory in sonic-buildimage while jenkins testing of sonic-utilities.
We need to create build dependency on sonic_yang_models PKG too.

I was referring yang models using relative path first, which is not needed after this PR.
```
pchaudha@server05:/home/pchaudha/srcCode/azure_myforks/sonic-buildimage/src/sonic-utilities$ git diff
diff --git a/sonic-utilities-tests/config_mgmt_test.py b/sonic-utilities-tests/config_mgmt_test.py
index d8704aa..aec7f75 100644
--- a/sonic-utilities-tests/config_mgmt_test.py
+++ b/sonic-utilities-tests/config_mgmt_test.py
@@ -15,7 +15,6 @@ class TestConfigMgmt(TestCase):
     '''

     def setUp(self):
-        config_mgmt.YANG_DIR = "./../../../sonic-yang-models/yang-models/"
         config_mgmt.CONFIG_DB_JSON_FILE = "startConfigDb.json"
         config_mgmt.DEFAULT_CONFIG_DB_JSON_FILE = "portBreakOutConfigDb.json"
         return
```

Signed-off-by: Praveen Chaudhary pchaudhary@linkedin.com



**- Why I did it**
Since we can not refer a directory in sonic-buildimage while jenkins testing of sonic-utilities.
We need to create build dependency on sonic_yang_models PKG too.

**- How I did it**
create build dependency on sonic_yang_models PKG too.

**- How to verify it**
Build the PKG locally, I see yang-models were installed at and tests run was successful.
```
pchaudha@server05:/home/pchaudha/srcCode/azure_myforks/sonic-buildimage$ ls -l target/python-debs/python-sonic-utilities_1.2-1_all.deb
-rw-r--r-- 1 pchaudha pchaudha 170660 Jun 29 09:58 target/python-debs/python-sonic-utilities_1.2-1_all.deb
pchaudha@server05:/home/pchaudha/srcCode/azure_myforks/sonic-buildimage$
```

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
